mlir/test/Dialect/Linalg/transform-lower-pack.mlir

```diff
@@ -634,7 +645,6 @@ module attributes {transform.with_named_sequence} {
   transform.named_sequence @__transform_main(%module_op: !transform.any_op {transform.readonly}) {
     %unpack = transform.structured.match ops{["tensor.unpack"]} in %module_op
       : (!transform.any_op) -> !transform.op<"tensor.unpack">
-    // expected-error @below {{cannot lower to transpose + collapse + extract}} 
```

Comment:
If this is not an error any more, then need IR check to validate output.

Reply:
The IR check is above the `unpack_with_outer_dims_perm` function (starting on line 625). Am I missing something else?